AlphaWallet/Protection/Views/SplashView.swift

```diff
@@ -3,16 +3,31 @@
 import UIKit
 
 class SplashView: UIView {
+    let logoImageView = UIImageView(image: R.image.launch_icon())
+
     init() {
         super.init(frame: CGRect.zero)
+
         backgroundColor = .white
-        let logoImageView = UIImageView(image: R.image.launch_icon())
         logoImageView.translatesAutoresizingMaskIntoConstraints = false
         addSubview(logoImageView)
-        NSLayoutConstraint.activate([
-            logoImageView.centerXAnchor.constraint(equalTo: centerXAnchor),
-            logoImageView.centerYAnchor.constraint(equalTo: centerYAnchor),
-        ])
+        self.layoutSubviews()
+    }
+    override func layoutSubviews() {
+        if UIDevice.current.hasNotch {
+                   let guide = safeAreaLayoutGuide
```

Comment:
Indentation. Ca you fix the indentation for the entire commit?

Reply:
ok